include/mips/signal.h

```diff
@@ -55,17 +56,11 @@ typedef int sig_atomic_t;
  * a non-standard exit is performed.
  */
 #if defined(_LIBC) || defined(_KERNEL)
+
 struct sigcontext {
```

Comment:
Why can't we completely remove `sigcontext`? It looks artificial to have it defined like that.

Reply:
We can :)